test/jdk/jdk/crac/ContextOrderTest.java

```diff
@@ -146,6 +144,11 @@ private static void assertWaits(Runnable runnable, String cls, String method) th
                         // clears the flag without rethrowing it is a bug.
                         thread.interrupt();
                         thread.join(TimeUnit.NANOSECONDS.toMillis(deadline - System.nanoTime()));
+                        if (thread.getState() == Thread.State.WAITING) {
```

Comment:
I think you wanted to remove printing the stack traces here, too.

Reply:
Let's have it there. This is on the failure path, so this won't appear until the failure. And everytime the failure happens, it's  really required to know what is the stack trace of the thread that does not respond to the interruption.